addons/slack/template/index.js

```diff
@@ -3,6 +3,14 @@ const health = require('./health.js');
 const Sdk = require('@fusebit/add-on-sdk');
 const { WebClient } = require('@slack/web-api');
 
+async function getStorageClient(ctx) {
+  return await Sdk.createStorageClient(
+    ctx,
+    ctx.fusebit.functionAccessToken,
+    `boundary/${ctx.boundaryId}/function/${ctx.functionId}`
```

Comment:
I think this should be `boundary/${ctx.boundaryId}/function/${ctx.functionId}/root` (or some other "default" document), since the ctx.fusebit.functionAccessToken should have storage:* scoped to resource `/account/{{accountId}}/subscription/{{subscriptionId}}/storage/boundary/{{boundaryId}}/function/{{functionId}}/` (notice the trailing slash).

Reply:
I verified and this works as-is even with the trailing slash included in the storage resource path in `functionPermissions`